server_tests/test/clientTests.js

```diff
@@ -843,194 +841,178 @@ describe("RUST API TEST SUITE", function() {
             return testClient.startSession()
             .then( function onSuccess (res) {
                 sessionId = res.body.results.sessionId;
-                Comparison.oscSessionOpOutput(res, {'sessionId': sessionId});
+                validate.done(res.body, schema.names.commandStartSession);
```

Comment:
To be consistent place this before the `sessionId` assignment. Validation should be the first thing done in most of these test functions.